tests/unit/test_utils.py

```diff
@@ -573,8 +573,9 @@ def names():
         assert not any(n == name for n in names())
 
         # Check the first group are correct
-        assert all(x == y for x, y in
-                   zip(some_names, [c + name[1:] for c in chars]))
+        for x, y in zip(some_names, ['~' + name[1:]] +
+                ['~' + c + name[2:] for c in chars]):
```

Comment:
The linter doesn't like this (neither do I). I'd suggest assigning the second parameter to `zip` a variable and do `zip(some_names, expected_names)` or similar.